projects/schematics/src/add-spartacus/schema.json

```diff
@@ -27,6 +27,11 @@
       "type": "string",
       "description": "Configure feature level for application.",
       "default": "1.1"
+    },
+    "overwriteAppComponent": {
+      "type": "boolean",
+      "description": "Overwrite whole app.component.html file.",
+      "default": true
```

Comment:
I would vote for making it disabled by default. We probably should assume, that customer can run schematic at any point in time.

Reply:
Good point with default val. Fixed.